src/static/index.html

```diff
@@ -32,6 +32,7 @@
     <script type="text/javascript" src="assets/js/timingDiagram.js"></script>
     <!-- end IGV -->
     <script type="text/javascript" src="assets/js/typeahead.bundle.js"></script>
+    <script type="text/javascript" src="assets/js/any-resize-event.min.js"></script>
```

Comment:
Other imported JS seems to work.  All this does is add some magic to allow resize events on any element.